tools/editor/editor_node.cpp

```diff
@@ -6267,11 +6294,71 @@ EditorNode::EditorNode() {
 
 EditorNode::~EditorNode() {
 
-
+        
 	memdelete( EditorHelp::get_doc_data() );
 	memdelete(editor_selection);
+        memdelete(editor_plugins_over);
 	memdelete(file_server);
 	EditorSettings::destroy();
 }
+/*
+ * EDITOR PLUGIN LIST
+ */
+
+
+void EditorPluginList::make_visible(bool p_visible) {
+    if (!plugins_list.empty()) {
+        for (int i = 0; i < plugins_list.size(); i++) {
+            plugins_list[i]->make_visible(p_visible);
+        }
+    }
+}
+
+void EditorPluginList::edit(Object* p_object) {
+    if (!plugins_list.empty()) {
+        for (int i = 0; i < plugins_list.size(); i++) {
+            plugins_list[i]->edit(p_object);
+        }
+    }
+}
+
+bool EditorPluginList::forward_input_event(const InputEvent& p_event) {
+    bool discard = false;
+    if (!plugins_list.empty()) {
+        for (int i = 0; i < plugins_list.size(); i++) {
+            if (plugins_list[i]->forward_input_event(p_event)) {
+                discard = true;
```

Comment:
My idea is to keep them forwarding. So, all events on the list can process the events, if only a single returns true to discard, so the event is discarded on https://github.com/godotengine/godot/pull/4174/files/aedb507004bbab041d989956f012a4fa1805e5fb#diff-2da0993e2c70df7aafac2ee92cb5f366R816

The goal was, all the plugins can work together as a single full featured plugin.